packages/docs/docs/guides/5-request-client/1-payment-networks.md

```diff
@@ -5,42 +5,54 @@ keywords: Request Client, Payment networks
 
 ---
 
-# TODO
+When a user creates and sends a request, he can expect to receive the correct amount of money. But how does he keep track of the payments due and received? Request is blockchain agnostic, Request is, by itself, a ledger that documents request's information but it doesn't interact directly with any external blockchains.
 
-When a user creates and sends a request, he can expect to receive the correct amount of money. But how does he keep track of the payments due and received?
+There are different methods available for the requester and payer to agree on the payment status, and that is when payment networks come into play. A payment network is a specific context on how to agree on the payment status of a specific request.
```

Comment:
`Requester` => `payee`  

We should avoid to introduce synonym to avoid confusion

Reply:
`A payment network is a specific context` => `A payment network is a set of rules ...`